lib/api/shared/src/main/scala/net/wiringbits/api/models/SendVerifyEmail.scala

```diff
@@ -0,0 +1,13 @@
+package net.wiringbits.api.models
+
+import net.wiringbits.common.models.Email
+import play.api.libs.json.{Format, Json}
+
+object SendVerifyEmail {
+    case class Request(email: Email)
```

Comment:
This is a potential spam endpoint, hence, include a captcha, we already have everything wired for that